lib/storage/client.js

```diff
@@ -1543,5 +1544,81 @@ class Client extends CryptoConsumer {
         )
     }
   }
+
+   /**
+   * fetchSubscriptionsToComputations fetches all Computations the TozStore Client is registered for
+   */
+    async fetchSubscriptionsToComputations(subscription) {
+      const response = await this.authenticator.tsv1Fetch(
+        `${this.config.apiUrl}/v1/secure-compute/subscription?client_id=${subscription.ToznyClientID}`,
+        {
+          method: 'GET',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+        }
+      )
+      switch (response.status) {
+        case 201: {
+          let json = await response.json()
+          return Computation.decode(json)
+        }
+        case 403:
+            throw new Error('Unauthorized')
+        default:
+          throw new Error('Error fetching computations.')
+      }
+    }
+
+    /**
+     * fetchAvailableCOmputations fetches all available computations to the TozStore Client
+     */
+    async fetchAvailableComputations() {
```

Comment:
feel free to push this to master to unblock matthew but can you add documentation to the read me!